### DISCUSSION TITLE
I want to help but I'm not sure how to start

### DISCUSSION THREAD

**Discussion by VFViotti** [athom-tech/athom-configs#93] [General]
Hello to all,

I'm somewhat new to the Home Assistant environment, and I choose buying devices from Athom due to their open source approach of already shipping their devices with opensource firmware. Originally I purchased my switches with Tasmota, but after messing around for almost 2 years trying to find my way, I got bored of using Tasmota as being more rigid than I anticipated.

I recently performed the migration process to esp home, but being honest it was a big challenge for me, as the guidance was extremely spartan.

I would like to contribute, but I'm a total beginner in  ESP home, so at this moment I'm not comfortable in messing with the yaml configurations, but I would like to know if I can provide some work and expand on the instructions with information and details I needed and that will likely assist other beginners in the future.

If this is something that the contributors think would be a good idea? Is there any work for something like this already in development?

Thank you all, for your attention.

**Comment by madberry** [comment]
Hello,

If you'd like to contribute to esphome joining the esphome community on https://esphome.io/ would be a good thing to start with. 

At the bottom of the page you'll find the "join community" link which will take you to the discord server. This is also the place for any questions you might have about esphome.

**Reply by VFViotti** [reply]
Thank you. I'll look into it.
